nnvm/tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -925,39 +925,69 @@ def _get_sample(data, state):
         np.testing.assert_allclose(tf_samples, tvm_samples, rtol=1e-5, atol=1e-5)
         assert(tvm_sample_str == tf_sample_str)
 
+
 #######################################################################
-# LRN (Local Response Normalization)
-# ----------------------------------
+# Split
+# ------
+def _test_split(ip_shape, num_or_size_splits, axis):
+    tf.reset_default_graph()
+    dtype = 'float32'
+    in_data = tf.placeholder(dtype, ip_shape, name="in_data")
+    tf.split(in_data, num_or_size_splits, axis=axis, name="split")
+    np_data = np.random.uniform(size=ip_shape).astype(dtype)
+    with tf.Session() as sess:
+        final_graph_def = tf.graph_util.convert_variables_to_constants(
+            sess,
+            sess.graph.as_graph_def(add_shapes=True),
+            ['split'])
+        tf_output = run_tf_graph(sess, [np_data], ['in_data:0'], 'split:0')
+        tvm_output = run_tvm_graph(final_graph_def, [np_data], ['in_data'],
+                                   tf_output.shape, dtype)
+        np.testing.assert_allclose(tf_output, tvm_output, atol=1e-5, rtol=1e-5)
```

Comment:
Split results in multiple outputs, hence  suggest to enhance run methods to return multiple outputs and compare the same.

Reply:
[Split](https://www.tensorflow.org/versions/r1.0/api_docs/cc/class/tensorflow/ops/split) returns the split result with list so comparing with List content result.

Reply:
run_tf_graph is not designed to result a list. Also we pass 'split:0' to return only first result.
run_tvm_graph can return list if output_shape, output_dtypes are lists.

Please check.

Reply:
Yeah missed scenario, will work on it.